src/Relude/Functor/Reexport.hs

```diff
@@ -15,9 +16,16 @@ module Relude.Functor.Reexport
        , module Data.Functor
        , module Data.Functor.Compose
        , module Data.Functor.Identity
+#if MIN_VERSION_base(4,12,0)
+        , module Data.Functor.Contravariant
+#endif
        ) where
 
 import Data.Bifunctor (Bifunctor (..))
 import Data.Functor (Functor (..), void, ($>), (<$>))
 import Data.Functor.Compose (Compose (..))
 import Data.Functor.Identity (Identity (..))
+
+#if MIN_VERSION_base(4,12,0)
+import Data.Functor.Contravariant (Contravariant (..), phantom, (>$<), (>$$<), ($<), Predicate (..), Comparison (..), Equivalence (..), Op (..))
```

Comment:
Let's also reexport the following useful functions:

* `defaultComparison`
* `defaultEquivalence`
* `comparisonEquivalence`

The names are unambiguous and they might be useful. Also, I see HLint rules for them in `hlint.dhall` file but those names weren't exported.

Though, it's possible to use `wrap (==)` or `wrap compare` using `wrap` from `Relude.Extra.Newtype` but still, it's a good idea to reexport common things.

Reply:
Oh, my bad.. I thought they would be exported by doing `(..)`
I'll mention them too!